demo/server/main.js

```diff
@@ -6,4 +6,7 @@ Meteor.startup(() => {
 });
 
 // setup ProseMeteor on server
-let proseMeteorServer = new ProseMeteorServer({ snapshotIntervalMs: 5000 });
+let proseMeteorServer = new ProseMeteorServer({
```

Comment:
config values are passed into `ProseMeteorServer` so the user of the package can override defaults